### ISSUE TITLE
Installed, but doesn't seem to do anything. 

### ISSUE THREAD

**Issue by jaepage** [roadhump/SublimeLinter-eslint#103]
I have SublimeLinter installed and many linters working (csslint, jshint, etc). I've temporarily removed jshint because I wanted to see how eslint compares. 

I've created a test `.js` file with obvious errors, and I'm not seeing any indicator lights. The bottom-right button says `JavaScript`. Quitting/relaunching Sublime Text 3 doesn't help. I can see `eslint` in the SublimeLinter settings file. I can see `which eslint` in Terminal. 

Have I missed something, or have I confused the use of this? Thank you.

**Comment by robmcguinness** [comment]
check your `SublimeLinter.sublime-settings`...mine had the eslint linter disabled:

```javascript
{
"linters": {
  "eslint": {
      "@disable": true, // should be FALSE
      "args": [],
      "excludes": []
    }
  }
}
```

**Comment by janaspage** [comment]
Mine is false: https://github.com/janaspage/configs/blob/master/Sublime%20Text/SublimeLinter.sublime-settings

**Comment by roadhump** [comment]
@janaspage Do you have `.eslintrc` settings file? ESLint does not apply any rules without configuration. Try `eslint --init` in command line.

**Comment by roadhump** [comment]
@janaspage I guess you can just try any option and disable-enable any specific rule later for your codestyle. ESLint has a good [documentation](http://eslint.org/docs/user-guide/configuring), all rules and specific presets are there. 

Global settings can be stored in `.eslintrc` in your home directory.

**Comment by janaspage** [comment]
So the radio button for `Where will your code run, Node or browser?` does *not* correspond to an exclusive-or setting in the config? The doc appears to accept `true` and/or `false` for either/both.

**Comment by HyperSprite** [comment]
@robmcguinness I know your answer didn't help the OP but I just spent hours reading/searching/messing around with this until I found your answer. Thanks!

**Comment by thebinarypenguin** [comment]
@jaepage Did you ever get this solved? I think you might have the same issue as me #117

**Comment by souljacker** [comment]
Had the same issue, this helped me.

Perhaps eslint --init should me mentioned at the instructions?
